.vsts-ci.yml

```diff
@@ -15,154 +16,163 @@ pr:
-            parallel: true
-  
-      - job: OSX_latest
-        displayName: 'Mac OS'
-        pool:
-          vmImage: 'macOS-latest'
-        strategy:
-          matrix:
-            ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
-              Debug:
-                _BuildConfig: Debug
-                _SignType: none
-                _DotNetPublishToBlobFeed : false
-            Release:
-              _BuildConfig: Release
-              _SignType: none
-              _DotNetPublishToBlobFeed : false
-        variables:
-        - name: _RID
-          value: osx-x64
-        steps:
-        - checkout: self
-          clean: true
-        - script: eng/common/cibuild.sh
-            --configuration $(_BuildConfig)
-            --prepareMachine
-            /p:RID=$(_RID)
-          displayName: Build
-        - task: ArchiveFiles@2
-          condition: and(eq(variables['system.pullrequest.isfork'], false), eq(variables['_BuildConfig'], 'Release'))
-          inputs:
-            rootFolderOrFile: 'artifacts/layout/dotnet-core-uninstall/'
-            includeRootFolder: false
-            archiveType: 'tar'
-            tarCompression: 'gz'
-            archiveFile: '$(Build.ArtifactStagingDirectory)/dotnet-core-uninstall.tar.gz'
-            replaceExistingArchive: true
-        - task: PublishBuildArtifacts@1
-          condition: and(eq(variables['system.pullrequest.isfork'], false), eq(variables['_BuildConfig'], 'Release'))
-          inputs:
-            pathtoPublish: '$(Build.ArtifactStagingDirectory)'
-            artifactName: 'drop-osx'
-            publishLocation: 'Container'
-            parallel: true
+      - template: /eng/common/templates-official/jobs/jobs.yml@self
+        parameters:
+          enableMicrobuild: true
+          enablePublishUsingPipelines: true
+          enablePublishBuildArtifacts: true
+          enablePublishTestResults: false
+          enableTelemetry: true
+          helixRepo: dotnet/cli-lab
+          timeoutInMinutes: 180 # increase timeout since BAR publishing might wait a long time
+          jobs:
+          - job: Windows
+            pool:
+              # For public or PR jobs, use the hosted pool.  For internal jobs use the internal pool.
+              # Will eventually change this to two BYOC pools.
+              ${{ if ne(variables['System.TeamProject'], 'internal') }}:
+                name: $(DncEngPublicBuildPool)
+                image: 1es-windows-2022-open
+                os: windows
+              ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+                name: $(DncEngInternalBuildPool)
+                image: 1es-windows-2022
+                os: windows
+            variables:
+            - name: _RID
+              value: win-x86
+            - name: _BuildConfig
+              value: Release
+            - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
All the public and PR code can probably be removed later as well since we're going into a split pipeline mode.

Reply:
MiYanni suggested yesterday that we should keep the split pipelines in the hope that we'd eventually be merging them with the -pr version again anyway.

Reply:
I intentionally left all public/internal logic in my PR too. I would not recommend removing it as there is a likelihood of wanting it if we need to do 1ES templates for public pipelines too in the future.

Reply:
ahh, makes sense. I think he's optimistic but it is safer to leave this in place for now just in case.